crates/sound/src/audio_backends/pw_backend.rs

```
@@ -415,4 +415,30 @@ impl AudioBackend for PwBackend {
         self.thread_loop.unlock();
         Ok(())
     }
+
+    fn start(&self, stream_id: u32) -> Result<()> {
+        println!("pipewire start");
+        self.thread_loop.lock();
+        let stream_hash = self.stream_hash.read().unwrap();
+        if let Some(stream) = stream_hash.get(&stream_id) {
+            stream.set_active(true).expect("could not start stream");
+        } else {
+            error!("stream not found for the given stream_id");
```

same as earlier, should probably return an error